doc/example/celer-sim.rst

```diff
@@ -0,0 +1,10 @@
+.. Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+.. See the doc/COPYRIGHT file for details.
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _example_celer_sim:
+
+Celeritas EM for profiling
```

Comment:
Since we're expanding celer-sim to include optical, and likely more in the near future, it's more of a Celeritas standalone for profiling.